lib/core/configuration.dart

```diff
@@ -560,6 +561,18 @@ class Configuration {
     }
   }
 
+  int getAlbumGridSize() {
+    if (_preferences.containsKey(keyAlbumGridSize)) {
```

Comment:
Can move it to LocalPref, same place where we are storing sort order? Config is already too crowded